packages/mdc-select/mdc-select.scss

```diff
@@ -177,3 +177,66 @@
     }
   }
 }
+
+/* postcss-bem-linter: define multi-select */
+
+/* stylelint-disable plugin/selector-bem-pattern */
+.mdc-multi-select {
+  appearance: none;
+  width: 250px;
+  padding: 0;
+  border: 1px solid rgba(black, .1);
```

Comment:
here you'll want to use the `border-color` property in conjunction with [mdc-theme-prop](https://github.com/material-components/material-components-web/tree/master/packages/mdc-theme#mdc-theme-prop-mixin) rather than a hard-coded value. You'll also want to account for dark themes. This is a general rule of thumb when working with colors in components.

```scss
.mdc-multi-select {
  // ...
  border: 1px solid;

  @include mdc-theme-prop(border-color, text-divider-on-light);

  @include mdc-theme-dark {
    @include mdc-theme-prop(border-color, text-divider-on-dark);
  }
}
```